db/data/20230120102649_fix_uwe_trainees.rb

```diff
@@ -0,0 +1,120 @@
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Mon, 05 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Fri, 16 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Mon, 19 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Thu, 15 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Wed, 19 Oct 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Wed, 31 Aug 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Mon, 24 Jan 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Mon, 05 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Mon, 19 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Fri, 09 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Thu, 11 Aug 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Mon, 05 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Tue, 27 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Mon, 19 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Fri, 09 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Tue, 27 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::PERSONAL_REASONS, withdraw_date: "Mon, 25 Jul 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Sun, 18 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Fri, 17 Jun 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Mon, 03 Oct 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Wed, 14 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Mon, 05 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Mon, 05 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Mon, 05 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::UNKNOWN, withdraw_date: "Mon, 05 Sep 2022 00:00:00 +0000" },
+        { withdraw_reason: WithdrawalReasons::PERSONAL_REASONS, withdraw_date: "Mon, 04 Apr 2022 00:00:00 +0000" },
+      ]
+
+    withdrawals_slugs.each_with_index do |slug, index|
+      trainee = Trainee.find_by(slug[:slug])
```

Comment:
@richardpattinson this won't work as it is expecting an id integer.

You need something like this:
```ruby
Trainee.find_by(slug: slug[:slug])
```